src/main/java/org/wildfly/security/http/impl/DigestAuthenticationMechanism.java

```diff
@@ -413,4 +428,11 @@ private void fail() throws AuthenticationMechanismException {
             throw httpDigest.mechCallbackHandlerFailedForUnknownReason(t);
         }
     }
+
+    private static String decodeRfc2231(String encoded) throws UnsupportedEncodingException {
```

Comment:
Just idea. Isn't this good candidate for pulling to common place, e.g. wildfly-common?

Reply:
as I dont see more places where such encoding would be used and there is no relation to existing wildfy-common utils, I would prefer to keep it here and private